GVRf/Framework/framework/src/main/java/org/gearvrf/scene_objects/GVRVideoSceneObject.java

```diff
@@ -254,8 +255,9 @@ public long getTimeStamp() {
 
     private static class GVRVideo implements GVRDrawFrameListener {
 
+        private final GVRContext mContext;
         private SurfaceTexture mSurfaceTexture = null;
```

Comment:
The surfaceTexture can still be leaked because the GVRVideo instance can be leaked.